lib/data/external/AwsClient.js

```diff
@@ -43,7 +43,10 @@ class AwsClient {
                     const log = createLogger(reqUids);
                     log.error('err from data backend',
                     { error: err, dataStoreName: this._dataStoreName });
```

Comment:
While we are at error management, we may want to log `err.message` instead, here and below, as @rahulreddy told me this is what elastic search expects, and also error messages are often generated so they will not appear when logging the error object.